addons/mass_mailing/models/res_company.py

```diff
@@ -10,9 +10,9 @@ class ResCompany(models.Model):
     def _get_social_media_links(self):
         self.ensure_one()
         return {
-            'social_facebook': self.social_facebook,
-            'social_linkedin': self.social_linkedin,
-            'social_twitter': self.social_twitter,
-            'social_instagram': self.social_instagram,
-            'social_tiktok': self.social_tiktok,
+            'social_facebook': "https://www.facebook.com/Odoo",
+            'social_linkedin': "https://www.linkedin.com/company/odoo",
+            'social_twitter': "https://twitter.com/Odoo",
+            'social_instagram': "https://www.instagram.com/explore/tags/odoo/",
+            'social_tiktok': "https://www.tiktok.com/@odoo_official",
```

Comment:
If we are going to keep company specific fields then we should keep this code as it is.